Rarime/Code/Constants/Videos.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+enum Videos {
+    static let removeCase = Bundle.main.url(forResource: "RemoveCase", withExtension: "mp4")!
+    static let scanMrzPassport = Bundle.main.url(forResource: "ScanMrzPassport", withExtension: "mp4")!
+    static let readNfcPassport = Bundle.main.url(forResource: "ReadNfcPassport", withExtension: "mp4")!
```

Comment:
let's remove "Passport" suffix, it was needed to differ between id card and external passport